content/post/52-go-ipfs-0.4.18.md

```diff
@@ -0,0 +1,283 @@
+---
+date: 2018-11-01
+url: 52-go-ipfs-0.4.18
+title: go-ipfs 0.4.18 released
+author: Steven Allen and Erik Ingenito
+---
+
+[go-ipfs 0.4.18](https://dist.ipfs.io/#go-ipfs) has been released! This is one of largest go-ipfs releases to date; 3 months in the making. *Thanks to all our contributors for your awesome work!*
+
+## Highlights
+The headline features this release are experimental [QUIC support](#quic), the [gossipsub](#pubsub)
+pubsub routing algorithm, [pubsub message signing](#pubsub), a refactored `ipfs p2p`
+[command](#commands-changes), and a new [WebUI](#webui). However, that's just scratching the surface.
+
+
+## QUIC
+First up, on the networking front, this release has also introduced experimental
+support for the QUIC protocol. QUIC is a new UDP-based network transport that
+solves many of the long standing issues with TCP.
+
+For us, this means (eventually):
+
+* **Fewer local resources**. TCP requires a file-descriptor per connection while
+  QUIC (and most UDP based transports) can share a single file descriptor
+  between all connections. This should allow us to dial faster and keep more
+  connections open.
+* **Faster connection establishment**. When client authentication is included, QUIC
+  has a three-way handshake like TCP. However, unlike TCP, this handshake brings
+  us all the way from 0 to a fully encrypted, authenticated, and
+  multiplexed connection. In theory (not yet in practice), this should
+  significantly reduce the latency of DHT queries.
```

Comment:
...meaning that applications like XX (DHT heavy) should run much faster!

Reply:
Unfortunately I don't know about 'much faster'. Maybe for adding files.

Reply:
I think this sentence should have an addendum that clarifies who benefits from faster DHT queries